tools/dotcms-cli/cli/src/test/java/com/dotcms/cli/common/ExceptionHandlerTest.java

```diff
@@ -44,32 +46,42 @@ void Test_Handle_WebApplication_Exception() {
         NotFoundException noise = new NotFoundException("No pineapple Flavor today");
         Exception handled = exceptionHandler.handle(noise);
         Assertions.assertTrue(handled instanceof WebApplicationException);
-        Assertions.assertTrue(handled.getMessage().contains(config.messages().get(404)));
+        // On recent versions of Quarkus, the custom message is set as the reason phrase of the response
+        //WebApplications have an immutable message so we can't change it. 404 will always be Not Found etc...
+        BuiltResponse response = (BuiltResponse) ((WebApplicationException) handled).getResponse();
+        //Therefore the custom message needs to be extracted from the response
+        Assertions.assertTrue(response.getReasonPhrase().contains(config.messages().get(404)));
+        //Assertions.assertTrue(handled.getMessage().contains(config.messages().get(404)));
```

Comment:
please remove those commented lines. It will make the code cleaner

Reply:
done